globaldog.py

```diff
@@ -63,7 +65,7 @@ def handleErrors(datadogResponse):
 screenboard["board_title"] = "{} {}".format(
```

Comment:
```python
print("Sending updated screenboard to Datadog...")
```

It seems "pint" was likely a typo, and reverting it to "print" is necessary to ensure expected behavior.